### PR TITLE
Initial implementation of experimental interop with Jetpack Compose

### PR DESCRIPTION
**Pull request by akurasov**

### CODE REVIEW

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt [hunk 1]
```diff
@@ -50,54 +54,92 @@ class ComposePlugin : Plugin<Project> {
                 // we want to avoid creating tasks like package, run, etc. to avoid conflicts with other plugins
                 configureApplicationImpl(project, desktopExtension.application)
             }
-        }
 
-        fun ComponentModuleMetadataHandler.replaceAndroidx(original: String, replacement: String) {
-            module(original) {
-                it.replacedBy(replacement, "org.jetbrains.compose isn't compatible with androidx.compose, because it is the same library published with different maven coordinates")
+            if (androidExtension.useAndroidX) {
+                println("useAndroidX is an axperimental feature at the moment!")
```
Comment (olonho) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt:59]: experimental

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt [hunk 2]
```diff
@@ -50,54 +54,92 @@ class ComposePlugin : Plugin<Project> {
...
+                    //
+                    // List of all org.jetbrains.compose libraries is here:
+                    // https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/
+                    //
+                    // (use ./gradle printAllAndroidxReplacements to know what dependencies should be here)
+                    //
+                    // It is temporarily solution until we will be publishing all MPP artifacts in Google Maven repository.
+                    // Or align versions with androidx artifacts and point MPP-android artifacts to androidx artifacts (is it possible?)
+                    it.replaceAndroidx(
+                        "androidx.compose.animation:animation",
```
Comment (olonho) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt:94]: Keep it in list and iterate over it.

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt (hunk 1 above)
Comment (akurasov) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt]: Fixed

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt (hunk 2 above)
Comment (akurasov) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt:95]: Done

**Review by igordmn — APPROVED** [review]

**Pull request merged**